backend/src/lang/intents/handlers/handler.help.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { IntentHandler } from './handler.abstract';
+import { DatabaseLangService } from 'connectors/database-lang.service';
+import { ConversationHistory } from 'database/conversationHistory/interfaces/conversationHistory.interface';
+import { ExplanationService, IGNORE_INTENTS } from '../../explanation/explanation.service';
+
+@Injectable()
+export class HelpIntentHandler extends IntentHandler{
+
+    constructor(private databaseLangService: DatabaseLangService, private explanationService: ExplanationService){
+        super();
+    }
+
+    /**
+     * Proccesses a given DialogFlow Reponse
```

Comment:
Da hat sich ein Buchstabe zu viel in den die Beschreibung geschlichen. (Finden wir automatisiert auch mit den kommenden JsDoc Linting rules nicht. Also lieber doch mal anmerken, wenn ich es schon sehe.)